roseau/load_flow/models/transformers/parameters.py

```diff
@@ -353,7 +339,6 @@ def from_power_factory(
         pc: float | Q_[float],
```

Comment:
I didn't notice this in your last PR but the parameters `uhv` and `ulv` for the `from_power_factory` method must not be renamed to `up` and `us`. The whole point of this method is that it uses the power factory notation and parameters and converts to them to the proper RLF object.